### ISSUE TITLE
Servo `./mach` fails from Gecko checkout, related to WPT

### ISSUE THREAD

**Issue by jryans** [servo/servo#17132]
STR:

1. Get a recent Gecko checkout (2017-06-01)
2. Run something like: `(cd servo; ./mach help)`

`mach` startup now fails because of missing WPT dependencies:

```
Pip failed to execute properly:
Output:
Error: Could not open requirements file: [Errno 2] No such file or directory: '/Users/jryans/projects/mozilla/gecko-dev-stylo/servo/../testing/web-platform/harness/requirements.txt'
```

I tried to adjust the paths in `servo/python/mach_bootstrap.py` for the `is_firefox` case, but this led to further errors about missing imports.

**Comment by jryans** [comment]
@jgraham I made a few attempts locally, but couldn't spot how to resolve the import errors.

**Comment by jryans** [comment]
I was able to work around this locally by commenting out the WPT parts of `servo/python/mach_bootstrap.py`, but of course a real fix would be best.

**Comment by jryans** [comment]
This appears to be fixed by #17163. Thanks @jgraham! 😁
